trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/rescale_non_dimensional.py

```diff
@@ -0,0 +1,133 @@
+## @ingroup Planform
+#rescale_non_dimensional.py
+
+# Created : May 2020, E. Botero
+# Modified:
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import SUAVE
+import numpy as np
+
+
+# ----------------------------------------------------------------------
+#  Set Origin Non-Dimensional
+# ----------------------------------------------------------------------
+
+def set_origin_non_dimensional(vehicle):
+    """ Places the origin of all components 
```

Comment:
Add more explanation here. It's not clear to me what this function is doing from the description.

Reply:
Done